Source/TrackableEntities.Client/EntityBase.cs

```diff
@@ -23,11 +23,24 @@ public abstract partial class EntityBase : INotifyPropertyChanged, ITrackable, I
             (Expression<Func<TResult>> property)
         {
             string propertyName = ((MemberExpression)property.Body).Member.Name;
+            NotifyPropertyChanged(propertyName);
+        }
+
+        /// <summary>
+        /// Fire PropertyChanged event.
+        /// </summary>
+        /// <param name="propertyName">Property name.</param>
+#if NET40
+        protected virtual void NotifyPropertyChanged(string propertyName)
+#else
+        protected virtual void NotifyPropertyChanged([System.Runtime.CompilerServices.CallerMemberName] string propertyName = "")
```

Comment:
I thought maybe you might want to keep the namespace inline since it's conditional, but OK.